dbt/adapters/spark/python_submissions.py

```diff
@@ -96,7 +106,7 @@ def _submit_job(self, path: str, cluster_spec: dict) -> str:
 
     def _submit_through_notebook(self, compiled_code: str, cluster_spec: dict) -> None:
         # it is safe to call mkdirs even if dir already exists and have content inside
-        work_dir = f"/dbt_python_model/{self.schema}/"
+        work_dir = f"/Shared/dbt_python_model/{self.schema}/"
```

Comment:
For workspace, I noticed that only admin user have access to root dir, but Shared can be accessed by any user, hence updated it. @lostmygithubaccount